### PR TITLE
Fix oncotree bug

### PR DESCRIPTION
**Pull request by allaway**
Whoops, I forgot to add the new tumor types in https://github.com/nf-osi/nf-metadata-dictionary/pull/103 as valid values... this fixes that.

### CODE REVIEW

**Comment by anngvu** [comment]
Hey, I implemented a different patch while I was adding "age". I noticed that the comma caused a problem, as you can see:

```
{
                    "@id": "bts:LowGradeGlioma"
                },
                {
                    "@id": "bts:NOS"
```
So I think what we'll have to do is use a label without a comma.

**Comment by allaway** [comment]
Oh, that's interesting. I am fairly sure we can have the comma if we wrap the value in quotes under `Valid Values`. If you are working on the same fix, I'll just close and wait for your PR. :)

File: NF.jsonld [hunk 1]
```diff
@@ -3549,26 +3552,50 @@
...
-                    "@id": "bts:Meningioma"
+                    "@id": "bts:Oligoastrocytoma"
                 },
                 {
-                    "@id": "bts:Normal"
+                    "@id": "bts:High-GradeGlioma"
                 },
                 {
-                    "@id": "bts:Teratoma"
+                    "@id": "bts:NOS"
```
Comment (anngvu) [inline NF.jsonld:3583]: Quoting doesn't work

Comment (allaway) [reply NF.jsonld:3583]: Interesting! I could have sworn that works, but obviously mistaken. Thanks for checking!

**Pull request closed without merging**